### DISCUSSION TITLE
Add a GRPC Java monorepo config

### DISCUSSION THREAD

**Discussion by danielcompton** [renovatebot/renovate#26623] [Suggest an Idea]
### Tell us more.

https://github.com/grpc/grpc-java has a monorepo of dependencies, it seems like it could be good to include it as a default monorepo source?

**Comment by secustor** [comment]
Do you consider contributing this yourself? 
See https://github.com/renovatebot/renovate/pull/26468 for example how to implement this.

**Reply by danielcompton** [reply]
Yup! https://github.com/renovatebot/renovate/pull/26631
